docs/rfcs/XeGPU.md

```diff
@@ -222,11 +222,12 @@ Attribute `chunk_size_per_lane` specifies the size being loaded per each work it
         	  tensor_desc<16xuint8, #Scattered>, vector<16xi1> into vector<16xuint8>
 ```
 
-When loading a tensor_desc with chunk_size_per_lane attribute, the output vector must be 2D vector, with the chunk being treated as a new dimension. The consecutive 1D tensor data being loaded can be viewed as a 2D tensor loaded with transposition, with the chunk dimension transposed to the outer dimension.
+When a tensor_desc with chunk_size_per_lane attribute is loaded, the output vector must be 2D vector, with the chunk being treated as a new dimension. The consecutive 1D tensor data being loaded can be viewed as a 2D tensor loaded with transposition, with the chunk dimension transposed to the outer dimension. The transpose=[1,0] attribute must be present when chunk_size_per_lane is larger than 1. 
```

Comment:
why special case for chunk_size > 1. we can just say transpose=[1, 0] must always be present. right?

Reply:
if chunk_size = 1, then there is no transpose effect.  Or you mean the result should be always 2d as [chunk_size, inner_dim]. So if chunk_size =1, the result should be [1x16] instead of [16]?  What is the benefit of doing this?

Reply:
yes. result is always 2d. the benefit is we don't need special verification code that needs to run for chunk_size == 1 case.

Reply:
OK. I just disallow the condition "chunk_size == 1" because that is useless.  User must not specify "chunk_size == 1".